docs/getting-started/install-configure.md

```diff
@@ -442,7 +442,9 @@ watch kubectl get pkg
 
 ```console
 # create service principal with Owner role
-az ad sp create-for-rbac --sdk-auth --role Owner > "creds.json"
+az ad sp create-for-rbac --name servicePrincipalname \
+                         --role Owner \
+                         --scopes /subscriptions/subcriptionId/resourceGroups/resourceGroupName > "creds.json"
```

Comment:
Adjusting as follows might help make it clear that folks are expected to replace the included values:
```suggestion
                         --scopes /subscriptions/[subcriptionId]/resourceGroups/[resourceGroupName] > "creds.json"
```